hdf5-sys/src/h5l.rs

```diff
@@ -182,32 +226,79 @@ extern "C" {
         n: hsize_t, buf: *mut c_void, size: size_t, lapl_id: hid_t,
     ) -> herr_t;
     pub fn H5Lexists(loc_id: hid_t, name: *const c_char, lapl_id: hid_t) -> htri_t;
-    pub fn H5Lget_info(
-        loc_id: hid_t, name: *const c_char, linfo: *mut H5L_info_t, lapl_id: hid_t,
+    #[cfg_attr(hdf5_1_12_0, deprecated(note = "deprecated in HDF5 1.12.0, use H5Lget_info2()"))]
+    #[cfg_attr(not(hdf5_1_12_0), link_name = "H5Lget_info")]
+    pub fn H5Lget_info1(
+        loc_id: hid_t, name: *const c_char, linfo: *mut H5L_info1_t, lapl_id: hid_t,
     ) -> herr_t;
+    #[cfg(hdf5_1_12_0)]
+    pub fn H5Lget_info2(
+        loc_id: hid_t, name: *const c_char, linfo: *mut H5L_info2_t, lapl_id: hid_t,
+    ) -> herr_t;
+    #[cfg_attr(
+        hdf5_1_12,
```

Comment:
@mulimoen Should this be `hdf5_1_12_0`? (here and in tons of other places)

Reply:
Well spotted!